docs/04-operation-guides/operations/ra-01-enable-kyma-with-runtime-agent.md

```diff
@@ -3,9 +3,9 @@ title: Enable Kyma with Runtime Agent
 ---
 
 [Runtime Agent](../../01-overview/main-areas/application-connectivity/ra-01-runtime-agent-overview.md) is a component that is used in the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md) of Application Connectivity. 
-By default, Kyma uses the [legacy Application Connectivity mode](../../01-overview/main-areas/application-connectivity/README.md), which does not support integration with Compass.
+By default, Kyma uses the [standalone Application Connectivity mode](../../01-overview/main-areas/application-connectivity/README.md), which does not support integration with Compass.
 Because of this, on installation, you must:
-- Disable the components used in the legacy mode by setting the **global.disableLegacyConnectivity** value to `true`, and 
+- Disable the components used in the standalone mode by setting the **global.disableLegacyConnectivity** value to `true`, and 
```

Comment:
```suggestion
- Disable the components used in the standalone mode by setting the **global.disableLegacyConnectivity** value to `true`
```

Reply:
I want it to be clear that you need to do both, and that the code snippet below does both. I'll leave it as is.